### PR TITLE
update roadmap to include 2.3 plan and 2.4 early ideas

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change updates the 2.2.0 documentation with version 2.3.0 plans and some early thinking for 2.4.0

### CODE REVIEW

File: Documentation/Contributing/Roadmap.md [hunk 1]
```diff
@@ -23,54 +23,54 @@ The Mixed Reality Toolkit will require Unity 2018.4.
 
 > When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will update to the LTS release. MRTK will also support the latest non-beta (ex: 2019.1) tech branch version of Unity, at the time at which MRTK was released.
 
-### 2.2.0
+### 2.3.0
+
+The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overal plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
```
Comment (keveleigh) [inline Documentation/Contributing/Roadmap.md]:
```suggestion
The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overall plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
```

File: Documentation/Contributing/Roadmap.md [hunk 2]
```diff
@@ -83,6 +83,7 @@ Themes:
 
 Potential areas of focus
 
-- Mobile VR
-- Leap Motion
+- Leap Motion support (Unity 2018 and Unity 2010)
```
Comment (keveleigh) [inline Documentation/Contributing/Roadmap.md]:
```suggestion
- Leap Motion support (Unity 2018 and Unity 2019)
```

File: Documentation/Contributing/Roadmap.md [hunk 3]
```diff
@@ -23,54 +23,54 @@ The Mixed Reality Toolkit will require Unity 2018.4.
...
+
 **Developer education**
 
 [Developer documentation](https://microsoft.github.io/MixedRealityToolkit-Unity) and example scenes are, like stability, an ongoing priority for the MRTK team.
 
-**Mobile AR**
+**Unity 2019.3 XR SDK support**
 
-The Mixed Reality Toolkit will be embracing ARFoundation from Unity to add official support for mobile AR platforms (Android ARCore and iOS ARKit). Future MRTK releases will build upon this work to support Unity's XR SDK platform.
+XR SDK is Unity's new mixed reality platform. The Microsoft Mixed Reality Toolkit is committed to supporting our customers on this new platform. All new platform support in MRTK is be delivered via XR SDK.
```
Comment (keveleigh) [inline Documentation/Contributing/Roadmap.md]: > All new platform support

There may be times we spin up support for a platform not supported by XR SDK. Would we surface this as a custom plugin for XR SDK vs MRTK?

Comment (davidkline-ms) [reply Documentation/Contributing/Roadmap.md]: good question. i'll add "is expected to be"

**Pull request merged**